src/panels/lovelace/cards/hui-entity-card.ts

```diff
@@ -0,0 +1,235 @@
+} from "lit-element";
+
+import { applyThemesOnElement } from "../../../common/dom/apply_themes_on_element";
+import { computeStateName } from "../../../common/entity/compute_state_name";
+import { stateIcon } from "../../../common/entity/state_icon";
+
+import "../../../components/ha-card";
+import "../../../components/ha-icon";
+import "../components/hui-warning";
+
+import {
+  LovelaceCard,
+  LovelaceCardEditor,
+  LovelaceHeaderFooter,
+} from "../types";
+import { HomeAssistant } from "../../../types";
+import { fireEvent } from "../../../common/dom/fire_event";
+import { EntityCardConfig } from "./types";
+import { hasConfigOrEntityChanged } from "../common/has-changed";
+import { actionHandler } from "../common/directives/action-handler-directive";
+import { isValidEntityId } from "../../../common/entity/valid_entity_id";
+import { findEntities } from "../common/find-entites";
+import { createHeaderFooterElement } from "../create-element/create-header-footer-element";
+import { UNKNOWN, UNAVAILABLE } from "../../../data/entity";
+import { HuiErrorCard } from "./hui-error-card";
+
+@customElement("hui-entity-card")
+class HuiEntityCard extends LitElement implements LovelaceCard {
+  public static async getConfigElement(): Promise<LovelaceCardEditor> {
+    await import(
+      /* webpackChunkName: "hui-entity-card-editor" */ "../editor/config-elements/hui-entity-card-editor"
+    );
+    return document.createElement("hui-entity-card-editor");
+  }
+
+  public static getStubConfig(
+    hass: HomeAssistant,
+    entities: string[],
+    entitiesFill: string[]
+  ) {
+    const includeDomains = ["sensor", "light", "switch"];
+    const maxEntities = 1;
+    const foundEntities = findEntities(
+      hass,
+      maxEntities,
+      entities,
+      entitiesFill,
+      includeDomains
+    );
+
+    return {
+      entity: foundEntities[0] || "",
+    };
+  }
+
+  @property() public hass?: HomeAssistant;
+  @property() private _config?: EntityCardConfig;
+  private _footerElement?: HuiErrorCard | LovelaceHeaderFooter;
+
+  public setConfig(config: EntityCardConfig): void {
+    if (config.entity && !isValidEntityId(config.entity)) {
+      throw new Error("Invalid Entity");
+    }
+
+    this._config = config;
+
+    if (this._config.footer) {
+      this._footerElement = createHeaderFooterElement(this._config.footer);
+    }
+  }
+
+  public getCardSize(): number {
+    return 2;
+  }
+
+  protected render(): TemplateResult {
+    if (!this._config || !this.hass) {
+      return html``;
+    }
+
+    const stateObj = this.hass.states[this._config.entity];
+
+    if (!stateObj) {
+      return html`
+        <hui-warning
+          >${this.hass.localize(
+            "ui.panel.lovelace.warning.entity_not_found",
+            "entity",
+            this._config.entity
+          )}</hui-warning
+        >
+      `;
+    }
+
+    const showUnit = this._config.attribute
+      ? Boolean(stateObj.attributes[this._config.attribute])
```

Comment:
This will filter out if the value is falsey, like 0. 
```suggestion
      ? this._config.attribute in stateObj.attributes
```